.github/workflows/release.yml

```diff
@@ -55,21 +57,31 @@ jobs:
       - name: Update crate versions
         id: versioning
         run: |
+
+          # parse the release version from the branch name
+          #   `release-0.8.1` -> `0.8.1`
           release_version=$(echo ${{ github.ref_name }} | grep -oP '^release-\K\d+\.\d+\.\d+$')
```

Comment:
What is the `-P` flag to grep? The grep on my mac (`grep (BSD grep, GNU compatible) 2.6.0-FreeBSD`) doesn't have it :/

Reply:
It makes grep use the perl-compatible regex (PCRE) engine. I don't know why the default installation of grep on mac doesn't support it. You can install the homebrew version (`brew install grep`) which will be available as `ggrep` (https://stackoverflow.com/a/22704387), and does support it